test/scripts/imgtestlib.py

```diff
@@ -18,6 +18,11 @@
 SCHUTZFILE = "Schutzfile"
 OS_RELEASE_FILE = "/etc/os-release"
 
+# CI runner for common tasks.
+# Currently this is used for all gitlab CI jobs. In the future, we might switch to running build jobs on the same host
+# distro as the target image, but this CI runner will still be used for generic tasks like check-build-coverage.
+CI_RUNNER = "aws/fedora-40"
```

Comment:
A bit of a nitpick, but it would be IMHO nicer to have the `CI_RUNNER` defined in a configuration file, such as `Schutzfile`, instead of in a script.

Reply:
I considered it for a bit but was on the fence.  Since you brought it up, I'll do it.